test/test_multi_region_cubed_sphere.jl

```diff
@@ -92,28 +92,63 @@ function get_boundary_indices(Nx, Ny, Hx, Hy, ::North; operation=nothing, index=
     return range_x, Ny-Hy+1:Ny
 end
 
+# Solid body rotation
+R = 1        # sphere's radius
+U = 1        # velocity scale
+φʳ = 0       # Latitude pierced by the axis of rotation
+α  = 90 - φʳ # Angle between axis of rotation and north pole (degrees)
+ζᵣ(λ, φ, z) = - U * R * (sind(φ) * cosd(α) - cosd(λ) * cosd(φ) * sind(α))
```

Comment:
it's a bit misleading to call this solid-body rotation vorticity since it's the stream function

Reply:
call it `psi` please

Reply:
ψ

Reply:
Done.